docs/development/README.adoc

```diff
@@ -35,22 +87,83 @@ The `+initialize.sh+` script will:
 * increase authorization for `RandomBeacon` and `WalletRegistry` applications
 * register operator for `RandomBeacon` and `WalletRegistry` applications
 
-Please refer to the script's `help` for available command arguments and environment
-variables. `./install.sh --help` or `./initialize.sh --help`. Accounts unlocking
-and initialization can be run outside of these scripts as standalone hardhat tasks.
+For more info please refer to `./scripts/initialize.sh --help`
 
-To deploy and initialize contracts:
+```
+Usage: ./scripts/initialize.sh --network <network> --stake-owner <stake owner address> --staking-provider <staking provider address> --operator <operator address> --beneficiary <beneficiary address> --authorizer <authorizer address> --stake-amount <stake amount> --authorization-amount <authorization amount>
+
+Optional line arguments:
+
+        --network: Ethereum network for keep-core client. Available networks and settings are specified in the 'hardhat.config.ts'
+        --stake-owner: Stake owner address
+        --staking-provider: Staking provider address
+        --operator: Operator address
+        --beneficiary: Staking beneficiary address
+        --authorizer: Staking authorizer address
+        --stake-amount: Stake amount
+        --authorization-amount: Authorization amount
+```
 
-* run `./scripts/install.sh`. You need to run it only once.
+Afer executing `install.sh` and `initialize.sh` now you can run `./scripts/start.sh` to start a client. It will prompt you to chose from the available config files (that should match the one chosen for `./scripts/initialize.sh`) and the log level. 
```

Comment:
```suggestion
=== start.sh

Afer executing `install.sh` and `initialize.sh` now you can run `./scripts/start.sh` to start a client. It will prompt you to chose from the available config files (that should match the one chosen for `./scripts/initialize.sh`) and the log level. 
```